src/Core/Checkout/Order/Aggregate/OrderLineItem/OrderLineItemDefinition.php

```diff
@@ -86,7 +86,7 @@ protected function defineFields(): FieldCollection
             (new FkField('order_id', 'orderId', OrderDefinition::class))->addFlags(new ApiAware(), new Required()),
             (new ReferenceVersionField(OrderDefinition::class))->addFlags(new ApiAware(), new Required()),
             (new FkField('product_id', 'productId', ProductDefinition::class))->addFlags(new ApiAware()),
-            (new ReferenceVersionField(ProductDefinition::class))->addFlags(new ApiAware(), new Required()),
+            (new ReferenceVersionField(ProductDefinition::class))->addFlags(new ApiAware()),
```

Comment:
Hey, sorry but this is not possible. ReferenceVersionField has to be marked as required, otherwise it get's not filled correctly when writing, updating or merging the data.

Reply:
Thank you for your review, Oliver. It seems like I need to delve deeper into Shopware to fix this issue. I'll continue working on it.

Reply:
Hey @StoneKnocker are you going to work on this in the near future? Otherwise I would close this PR for now and you can feel free to open a new one, when you are ready.

Reply:
Fine, when I am able to address this issue, I will open a new PR.